test/uk/gov/hmrc/plasticpackagingtax/returns/models/request/JourneyActionSpec.scala

```diff
@@ -58,6 +58,7 @@ class JourneyActionSpec extends ControllerSpec {
         given(mockTaxReturnsConnector.find(refEq(taxReturnId))(any[HeaderCarrier])).willReturn(
           Future.successful(Right(Option(aTaxReturn(withId(taxReturnId)))))
         )
+        mockGetObligation()
```

Comment:
If you used an `afterEach()` hook and reset mocks there then you could do "common" mock setup in `beforeEach()`. No need for a change - just a suggestion for the future.